test/test_tracer.dart

```diff
@@ -80,15 +80,12 @@ class TestSpan implements Span {
   @override
   void setTag(String tagName, dynamic value) => tags[tagName] = value;
 
-  @override
-  set startTime(DateTime value) => startTime = value;
```

Comment:
`startTime` can't be used as a setter because it is final, that's why this setter was removed